<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaChartClass</class>
 <widget class="QMainWindow" name="CaChartClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CaChart</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>891</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Input Signal:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="SignalPathEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="OpenfileBtn">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="InputSignalBtn">
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="FilterBtn">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="FilterSlider">
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="singleStep">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="ProcessBtn">
            <property name="text">
             <string>Process</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>1061</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="proLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>1061</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mainLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1098</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInput">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionInput_Signal"/>
    <addaction name="actionSave_ProcessedSignal"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="actionAverage_Filter"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="actionProcess"/>
   </widget>
   <addaction name="menuInput"/>
   <addaction name="menuFilter"/>
   <addaction name="menuProcess"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionInput_Signal"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionProcess"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionMouse"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInput_Signal">
   <property name="icon">
    <iconset resource="CaChart.qrc">
     <normaloff>:/CaChart/icon/Resources/icon/open.png</normaloff>:/CaChart/icon/Resources/icon/open.png</iconset>
   </property>
   <property name="text">
    <string>Input Signal</string>
   </property>
  </action>
  <action name="actionAverage_Filter">
   <property name="text">
    <string>Filter</string>
   </property>
  </action>
  <action name="actionProcess">
   <property name="icon">
    <iconset resource="CaChart.qrc">
     <normaloff>:/CaChart/icon/Resources/icon/run.png</normaloff>:/CaChart/icon/Resources/icon/run.png</iconset>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="CaChart.qrc">
     <normaloff>:/CaChart/icon/Resources/icon/zoomin.png</normaloff>:/CaChart/icon/Resources/icon/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="CaChart.qrc">
     <normaloff>:/CaChart/icon/Resources/icon/zoomout.png</normaloff>:/CaChart/icon/Resources/icon/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="CaChart.qrc">
     <normaloff>:/CaChart/icon/Resources/icon/sweep.png</normaloff>:/CaChart/icon/Resources/icon/sweep.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionMouse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="CaChart.qrc">
     <normaloff>:/CaChart/icon/Resources/icon/hand1.png</normaloff>:/CaChart/icon/Resources/icon/hand1.png</iconset>
   </property>
   <property name="text">
    <string>Mouse</string>
   </property>
  </action>
  <action name="actionSave_ProcessedSignal">
   <property name="text">
    <string>Save ProcessedSignal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CaChart.qrc"/>
 </resources>
 <connections/>
</ui>
